modules/SofaSparseSolver/src/SofaSparseSolver/SparseLUSolver.h

```diff
@@ -68,13 +74,19 @@ class SparseLUSolver : public sofa::component::linearsolver::MatrixLinearSolver<
 
     Data<bool> f_verbose; ///< Dump system state at each iteration
     Data<double> f_tol; ///< tolerance of factorization
-
-    SparseLUSolver();
+    
     void solve (Matrix& M, Vector& x, Vector& b) override;
     void invert(Matrix& M) override;
 
+    SparseLUSolver();
+
 protected :
 
+    Data<bool> d_applyPermutation ;
+  
+    void fill_reducing_perm(cs A,int * perm,int * invperm);
```

Comment:
```suggestion
    void fill_reducing_perm(cs A, int * perm, int * invperm);
```
cosmetic: just put a space after a coma